### PR TITLE
add metrics and remove logs

### PR DESCRIPTION
**Pull request by ckr123**

### CODE REVIEW

File: domains/certificates/Query.API/API/MeasurementsSyncer/Metrics/MeasurementSyncMetrics.cs [hunk 1]
```diff
@@ -19,6 +19,16 @@ public class MeasurementSyncMetrics : IMeasurementSyncMetrics
     private long recoveredMeasurements = 0;
     private ObservableCounter<long> RecoveredMeasurements { get; }
```
Comment (sahma19) [inline domains/certificates/Query.API/API/MeasurementsSyncer/Metrics/MeasurementSyncMetrics.cs:21]: Kan vi få en total measurements metric også?

File: domains/certificates/Query.API/API/MeasurementsSyncer/Metrics/MeasurementSyncMetrics.cs [hunk 2]
```diff
@@ -19,6 +19,16 @@ public class MeasurementSyncMetrics : IMeasurementSyncMetrics
...
+    private long numberOfMeasurementsQuantityTooLow = 0;
+    private ObservableCounter<long> NumberOfMeasurementsQuantityTooLow { get; }
+
+    private long numberOfMeasurementsQuantityTooHigh = 0;
+    private ObservableCounter<long> NumberOfMeasurementsQuantityTooHigh { get; }
+
+    private long numberOfMeasurementsWrongQuality = 0;
+    private ObservableCounter<long> NumberOfMeasurementsWrongQuality { get; }
+
+
```
Comment (sahma19) [inline domains/certificates/Query.API/API/MeasurementsSyncer/Metrics/MeasurementSyncMetrics.cs:31]: kan vi få skrevet tests på at disse virker?

**Review by sahma19 — APPROVED** [review]

File: domains/certificates/Query.API/API/MeasurementsSyncer/Metrics/MeasurementSyncMetrics.cs (hunk 1 above)
Comment (ckr123) [reply domains/certificates/Query.API/API/MeasurementsSyncer/Metrics/MeasurementSyncMetrics.cs:21]: There already is ett_certificate_measurements_fetched

File: domains/certificates/Query.API/API/MeasurementsSyncer/Metrics/MeasurementSyncMetrics.cs (hunk 2 above)
Comment (ckr123) [reply domains/certificates/Query.API/API/MeasurementsSyncer/Metrics/MeasurementSyncMetrics.cs:31]: As we talked about we dont have any test right now and i think it is a lot of work to test a `+=` statement

**Review by martinhenningjensen — APPROVED** [review]

**Pull request merged**